### PR TITLE
Bugfix first round

### PR DESCRIPTION
**Pull request by jesus-vc**
Refactored to use switch statements for timeword app.

### CODE REVIEW

**Review by lorezi — APPROVED** [review]
LGTM!

File: bankly-app/models/user.js [hunk 1]
```diff
@@ -79,16 +77,26 @@ class User {
    *
    * */
 
+  //BUG are parameters needed here?
   static async getAll(username, password) {
```
Comment (lorezi) [inline bankly-app/models/user.js]: May I inquire why the user's password is being passed as a parameter?

**Review by lorezi — COMMENTED** [review]
LGTM!

File: bankly-app/models/user.js (hunk 1 above)
Comment (jesus-vc) [reply bankly-app/models/user.js]: @lorezi, these parameters were in the source code written by Springboard. I agree that they aren't needed. I'll remove them, ensure all tests are still passing, and merge.

**Pull request merged**